app/assets/stylesheets/global.css.scss

```diff
@@ -53,3 +53,50 @@ button.button a {
   color: white;
   border-radius: 18px;
 }
+
+.header {
+  // text-align: center;
+  background: asset-url("red-panda.jpg") no-repeat center center;
+  background-size: cover;
+  padding: 4rem;
+
+  .main-header {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.